android/app/src/main/res/values-kk/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <!-- TODO remove and put into build.gradle -->
+  <string name="key_server_upload_uri">https://exposure-2kabcv6c4a-uc.a.run.app/</string>
```

Comment:
This looks like a reminiscence from the repo split, might be best to delete the key from lokalise

Reply:
ok, done!